selfdrive/athena/athenad.py

```diff
@@ -101,6 +109,11 @@ def _do_upload(upload_item):
                         headers={**upload_item.headers, 'Content-Length': str(size)},
                         timeout=10)
 
+def _clear_log_batch_retry():
+  global current_num_retrieve_upload_log_batch
+  global last_uploaded_log_batch
```

Comment:
is this necessary?

Reply:
If we want to keep using those 2 global variables then we need the `global` keyword otherwise python will just create 2 local variables within the scope of the function therefore any modification to them would not be reflected on the global variables